Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -270,7 +273,7 @@ def is_probe_connected(client: Client, machine: str) -> dict:
 
 
 def query_processes_command(client: Client, args: dict):
-    machine = args.get('machine')
+    machine = str(args.get('machine'))
```

Comment:
No need to convert to a string as the demisto.args() returning all the arguments as str already. Did you encounter any issue with it?

Reply:
Yes, we were getting a MyPy error if we didn't added the typecast. Please let me know if you want us to remove it.